### PR TITLE
Create hosting aquarius, subgraph page

### LINKED ISSUE #1043: New Page: How to run and host own Aquarius
There are details on that here: https://github.com/oceanprotocol/aquarius/blob/main/developers.md

We can write an expanded step by step version that's easy for beginners to follow

### PR DESCRIPTION
**Pull request by akshay-ap**
Fixes #1043 #1058 

Changes proposed in this PR:

- [x] Create a guide on creating a server (using Digitalocean) along with installing docker compose. 
- [x] Create a guide on hosting Aquarius
- [x] Create a guide on hosting Ocean subgraph

### CODE REVIEW

**Review by jamiehewitt15 — CHANGES_REQUESTED** [review]
Currently this just covers running Aquarius, can you add an extra section on how to host it? Some info on the options and requirements for hosting would be great (e.g. AWS, Azure, Heroku etc) and it would be good if we could provide the steps for hosting for hosting on at least one of those services. Ideally we want to be able to help someone with zero backend knowledge to quickly get Aquarius hosted

**Comment by akshay-ap** [comment]
> Currently this just covers running Aquarius, can you add an extra section on how to host it? Some info on the options and requirements for hosting would be great (e.g. AWS, Azure, Heroku etc) and it would be good if we could provide the steps for hosting for hosting on at least one of those services. Ideally we want to be able to help someone with zero backend knowledge to quickly get Aquarius hosted

This is done here in a dedicated page: https://app.gitbook.com/o/mTcjMqA4ylf55anucjH8/s/xUK1Gh4cTFPYiiyojUxw/~/revisions/4bZs8MSwzxaSAjkMSawR/building-with-ocean/deploying-components/setup-server

I created a dedicated page because the setting server steps will be same for deploying Marketplace, Aquarius, Provider and Subgraph.

File: building-with-ocean/deploying-components/setup-server.md [hunk 1]
```diff
@@ -0,0 +1,65 @@
...
+
+![Configure droplet](<../../.gitbook/assets/image (8).png>)
+
+Also, select the region where you want Aquarius to be hosted and a root password.
+
+![](<../../.gitbook/assets/image (4).png>)
+
+![Click Create Droplet](<../../.gitbook/assets/image (7).png>)
+
+Finalize the parameters for the server, click on `Create Droplet.` After the server is ready, s`e`lect the Access console option from the dropdown.
```
Comment (jamiehewitt15) [inline building-with-ocean/deploying-components/setup-server.md:35]: The `e` in select is formatted wrong

File: building-with-ocean/deploying-components/setup-server.md [hunk 2]
```diff
@@ -0,0 +1,65 @@
...
+
+![Click Access Console](<../../.gitbook/assets/image (3).png>)
+
+![Click Launch Droplet Console](<../../.gitbook/assets/image (9).png>)
+
+A window will open with a terminal session. Now, the required infrastructure is ready for hosting Aquarius. Let's install docker and docker-compose on the server. Follow the installation guide [here](https://docs.docker.com/engine/install/ubuntu/).
```
Comment (jamiehewitt15) [inline building-with-ocean/deploying-components/setup-server.md:41]:
```suggestion
A window will open with a terminal session. Now, the required infrastructure is ready for hosting Aquarius, Provider or the Subgraph. Let's install docker and docker-compose on the server. Follow the installation guide [here](https://docs.docker.com/engine/install/ubuntu/).
```

**Review by jamiehewitt15 — APPROVED** [review]
Looks good to me

**Pull request merged**